File: bee1002.cs
// 1002 - Área do Círculo
// https://www.beecrowd.com.br/judge/pt/problems/view/1002

using System;

class URI
{

    static void Main(string[] args)
    {
        double pi = 3.14159, r;

        r = Convert.ToDouble(Console.ReadLine());

        System.Console.WriteLine("A=" + (pi * (r * r)).ToString("0.0000"));
    }
}

File: bee1002.java
// 1002 - Área do Círculo
// https://www.beecrowd.com.br/judge/pt/problems/view/1002

import java.util.Scanner;

public class bee1002 {

	public static void main(String[] args) {

		double area, raio, pi;
		Scanner entrada = new Scanner(System.in);

		raio = entrada.nextDouble();
		pi = 3.14159;
		area = pi * (raio * raio);

		System.out.printf("A=%.4f\n", area);
	}
}